anesthetic/plot.py

```diff
@@ -790,31 +768,30 @@ def kde_contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
                                            color=['c'],
                                            facecolor=['fc'],
                                            edgecolor=['ec']))
-    bw_method = kwargs.pop('bw_method', None)
-    xmin = kwargs.pop('xmin', None)
-    xmax = kwargs.pop('xmax', None)
-    ymin = kwargs.pop('ymin', None)
-    ymax = kwargs.pop('ymax', None)
+
+    if len(data_x) == 0 or len(data_y) == 0:
+        return np.zeros(0), np.zeros(0), np.zeros((0, 0))
```

Comment:
ditto

Reply:
ditto

Reply:
Removed in [6d0d8a3](https://github.com/williamjameshandley/anesthetic/pull/212/commits/6d0d8a3b823849e405569a0ea01bf17016f06064).